docs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "docs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@radix-ui/react-dropdown-menu": "^2.0.4",
+    "@types/node": "20.2.3",
+    "@types/react": "18.2.7",
+    "@types/react-dom": "18.2.4",
+    "autoprefixer": "10.4.14",
+    "class-variance-authority": "^0.6.0",
+    "eslint": "8.41.0",
+    "eslint-config-next": "13.4.3",
+    "lucide-react": "^0.221.0",
+    "next": "13.4.3",
+    "next-themes": "^0.2.1",
+    "postcss": "8.4.23",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "tailwind-merge": "^1.12.0",
+    "tailwindcss": "3.3.2",
+    "typescript": "5.0.4"
+  },
+  
+   "workspaces": {
```

Comment:
Not here, add it on root `package.json`. This file https://github.com/iamvishnusankar/next-sitemap/blob/master/package.json

Reply:
My bad got it now tho